modules/Bio/EnsEMBL/Compara/PipeConfig/Protists/ProteinTrees_conf.pm

```diff
@@ -0,0 +1,80 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::PipeConfig::Protists::ProteinTrees_conf
+
+=head1 SYNOPSIS
+
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Protists::ProteinTrees_conf \
+    -host mysql-ens-compara-prod-X -port XXXX
+
+=head1 DESCRIPTION
+
+The Protists PipeConfig file for ProteinTrees pipeline automating execution of homology-specific analyses.
+
+=cut
+
+package Bio::EnsEMBL::Compara::PipeConfig::Protists::ProteinTrees_conf;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Hive::Utils ('stringify');
+
+use base ('Bio::EnsEMBL::Compara::PipeConfig::ProteinTrees_conf');
+
+
+sub default_options {
+    my ($self) = @_;
+
+    return {
+        %{$self->SUPER::default_options},   # inherit the generic ones
+
+        'division'   => 'protists',
```

Comment:
I'm comparing files from this PR to those in `.../PipeConfig/Metazoa`. In Metazoa, after this line there is `'collection' => $self->o('division'),`. I'm aware it's not like that in all divisions. Why going without `'collection'` in Protists?

Reply:
This was a change in `e106` across divisions to make sure that the `collection/clusterset.name` will be `default` instead of `Protists`. You might remember the discussions we had as a team about it: [ENSCOMPARASW-4706](https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-4706)